haiku/build.rs

```diff
@@ -1,11 +1,20 @@
+use std::env;
+
 fn main() {
     cxx_build::bridge("src/lib.rs")
         .file("src/implementation.cc")
         .flag_if_supported("-std=c++11")
         .compile("tz_haiku");
 
-    println!("cargo:return-if-changed=include/interface.h");
-    println!("cargo:return-if-changed=src/implementation.cc");
     println!("cargo:return-if-changed=src/lib.rs");
-    println!("cargo:rustc-link-lib=be");
+    println!("cargo:return-if-changed=src/implementation.cc");
+    println!("cargo:return-if-changed=src/interface.h");
```

Comment:
Oops, thank you for catching that!

Reply:
thanks for commenting here. The build.rs directive is _rerun_ if changed, not return if changed. I pushed a new commit.

Reply:
Strange, I thought I copied the line instead of typing it out of memory. I have no idea where the typo came from.